cms/static/js/features/import/factories/import.js

```diff
@@ -76,7 +76,7 @@ define([
             $('#fileupload').fileupload({
                 dataType: 'json',
                 type: 'POST',
-                maxChunkSize: 20 * 1000000, // 20 MB
+                maxChunkSize: 10 * 1000, // 20 MB
```

Comment:
is this the correct chunk size?

Reply:
thanks for pointing it out. reverted this change, it was unintentional.